docs/developer-docs/latest/development/backend-customization/policies.md

```diff
@@ -1,164 +1,181 @@
+
+Each [route](/developer-docs/latest/development/backend-customization/routes.md) of a Strapi project can be associated to an array of policies. For example, a policy named `is-admin` could check that the request is sent by an admin user, and uses it for critical routes.
+<!-- TODO: remove this comment — the link above will work only when merged with PR #450 -->
 
-The policies are defined in each `./api/**/config/policies/` folders and plugins. They are respectively exposed through `strapi.api.**.config.policies` and `strapi.plugins.**.config.policies`. The global policies are defined at `./config/policies/` and accessible via `strapi.config.policies`.
+Policies can be global or scoped. [Global policies](#global-policies) can be associated to any route in the project. Scoped policies only apply to a specific [API](#api-policies) or [plugin](#plugin-policies).
 
-## How to create a policy?
+## Implementation
 
-There are several ways to create a policy.
+A new policy can be implemented:
 
-- Using the CLI `strapi generate:policy is-authenticated`.<br>Read the [CLI documentation](/developer-docs/latest/developer-resources/cli/CLI.md) for more information.
-- Manually create a JavaScript file named `is-authenticated.js` in `./config/policies/`.
+- with the [interactive CLI command `strapi generate`](/developer-docs/latest/developer-resources/cli/CLI.md#strapi-generate) 
+- or manually by creating a JavaScript file in the appropriate folder (see [project structure](/developer-docs/latest/setup-deployment-guides/file-structure.md)):
+  - `./src/policies/` for global policies
+  - `./src/api/[api-name]/policies/` for API policies
+  - `./src/plugins/[plugin-name]/policies/` for plugin policies
 
-**Path —** `./config/policies/is-authenticated.js`.
+<br/>
+Global policy implementation example:
 
 ```js
-module.exports = async (ctx, next) => {
-  if (ctx.state.user) {
-    // Go to next policy or will reach the controller's action.
+// path: ./src/policies/is-authenticated.js
+
+module.exports = (policyContext, { strapi }) => {
+  if (ctx.state.user) { // if a session is open
+    // go to next policy or reach the controller's action
     return await next();
   }
 
-  ctx.unauthorized(`You're not logged in!`);
+  ctx.unauthorized(`You're not logged in!`); // returns a 401 error
+
+  return true; // If you return nothing, Strapi considers you didn't want to block the request and will let it pass
 };
 ```
 
-In this example, we are verifying that a session is open. If it is the case, we call the `next()` method that will execute the next policy or controller's action. Otherwise, a 401 error is returned.
+`policyContext` is a wrapper arround the [controller](/developer-docs/latest/development/backend-customization/controllers.md) context. It adds some logic that can be useful to implement a policy for both REST and GraphQL.
+
+<br/>
+Policies can declare a `config` object to inject some custom configuration.
```

Comment:
Policies do not declare config but can be configure with a config

Reply:
```suggestion
Policies can be configured using a `config` object.
```